### DISCUSSION TITLE
Allow Users to Select Headless Mode for Systems with Displays Attached

### DISCUSSION THREAD

**Discussion by davidshq** [rustdesk/rustdesk#5751] [Feature Request]
One of the things I love about RDP is that a computer remains locked to local access while a remote session exists. The local user cannot see what is being done on the system, the screen remains locked.

I'm guessing this is done via a virtual desktop and I've seen that RustDesk supports Linux headless but as far as I can tell this only kicks in if there are no attached displays. If there is an attached display it will always prefer to use the display?

Could an option be added to allow individuals to choose whether they want to use a virtual monitor or not? When it is remoting into a system I'm using for myself rather than one that I'm supporting a client on, I always prefer a virtual display.

**Comment by vanavata** [comment]
I'm also trying to figure out how to set "virtual display 1" as the default option when I connect to the host.  Can someone please let me know if there is a way to configure this?

Thanks!
